src/Command/Ingester/LineItemIngesterCommand.php

```diff
@@ -110,7 +110,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                     'slug',
                     'startTimestamp',
                     'endTimestamp',
-                    'maxAttempts'
+                    'maxAttempts',
+                    'groupId'
```

Comment:
You can remove it from validation I think, it's optional to specify it in the csv. What do you think?

Reply:
I agree! Removed